### PR TITLE
Update tempto to 1.12

### PR DESCRIPTION
**Pull request by kokosing**

### CODE REVIEW

File: presto-product-tests/README.md [hunk 1]
```diff
@@ -225,10 +225,11 @@ setup outlined below:
     ./mvnw install -DskipTests
     ```
 
-2. Start Hadoop in pseudo-distributed mode in a Docker container:
+2. Start Presto dependant services as Docker containers:
 
     ```
-    presto-product-tests/conf/docker/singlenode/compose.sh up -d hadoop-master
+    presto-product-tests/conf/docker/singlenode/compose.sh up -d
```
Comment (arhimondr) [inline presto-product-tests/README.md]: I would rather specify all the required services explicitly here. I don't think we are going to add new ones to often. And anyway, if we added yet another one, documentation update would be required.

File: presto-product-tests/README.md [hunk 2]
```diff
@@ -237,17 +238,25 @@ setup outlined below:
...
+3. Add an IP-to-host mapping for the docker hosts in `/etc/hosts`.
 The format of `/etc/hosts` entries is `<ip> <host>`:
 
-    - On GNU/Linux add the following mapping: `<container ip> hadoop-master`.
-    The container IP can be obtained by running:
+    - On GNU/Linux run the following command add output of below command to `/etc/hosts`: 
 
         ```
-        docker inspect $(presto-product-tests/conf/docker/singlenode/compose.sh ps -q hadoop-master) | grep -i IPAddress
+        presto-product-tests/bin/hosts.sh singlenode
```
Comment (arhimondr) [inline presto-product-tests/README.md]: This script requires yet another external dependency. Personally i would prefer to have here simple instructions, like 

```
add

localhost mysql postgres
<container ip> hadoop-master

to /etc/hosts
```

But script has its own benefits. So, i don't have strong opinion here.

Also i know that this is not a scope of this PR, but i think we should change the ports of `mysql` and `postgree` from defaults to some specific. Mysql and Postgree ofthen are already installed on the host machine.  Than we should change the `mysql` and `postgres` hosts as well. Those names looks too generic.

**Comment by arhimondr** [comment]
lgtm

File: presto-product-tests/README.md (hunk 2 above)
Comment (kokosing) [inline presto-product-tests/README.md]: @maciejgrzybek Already did that as separate PR.

**Comment by kokosing** [comment]
Rebased and addressed comemnts.

@cberner Can you please take a look?

**Comment by martint** [comment]
Merged, thanks!

**Pull request closed without merging**